src/apps/halofi/common/halofi.game.constants.ts

```diff
@@ -36,9 +38,69 @@ export const transformRewardArrayToObject = (rewards: RewardBalance[]) => {
   return playerReward;
 };
 
-const ContractVersions = ['2.0.0', '2.0.1', '2.0.2', '2.0.3'];
+/**
+ * queue request and send request with support for throttling
+ * @param {any} request The given request to send
+ * @returns {Promise} A fulfilled promise after the request is processed
+ */
+const queue = new PQueue({ concurrency: 1, interval: 1000 });
```

Comment:
@meetkosiso can we please add the following adjustments to this config? Thanks
```suggestion
const queue = new PQueue({ interval: 1000, intervalCap: 4 });
```